### PR TITLE
chore: container disclaimer, add missing env for smooth setup

### PR DESCRIPTION
**Pull request by chrishollandaise**
Related to PR [#334](https://github.com/learnhouse/learnhouse/pull/334)

Changes:
1. Added container memory disclaimer to the docs to help circumvent build failures when compiling the frontend. [#392](https://github.com/learnhouse/learnhouse/issues/329)

2. Added the `NEXT_PUBLIC_LEARNHOUSE_TOP_DOMAIN` to the env sections.

Reason for the second change was I noticed that without this NextJS auth cookies would try to default to being secure, which won't work for `localhost`

### CODE REVIEW

**Comment by chrishollandaise** [comment]
One more thing, when I was setting up. I was confused by the inconsistencies for the env between the step 1 and 2 [frontend configuration section](https://docs.learnhouse.app/setup-dev-environment#frontend-configuration). 

I am wondering if this section should be condensed down to just include all the needed env variables for the setup? I didn't see any unique setup for when `NEXT_PUBLIC_LEARNHOUSE_DEFAULT_ORG` is set to test, and then set to default in the next step. That goes for the additional env variables that are included.

**Comment by swve** [comment]
Hey @chrishollandaise, nice finds and LGTM 👍

You're right about the extra step—it doesn't add much value and can definitely be combined with the following one. I originally included it to demonstrate how org detection works based on the 'Hosting Modes', but we can merge the two steps and include a brief note explaining that the multi-org setup won't work locally. 

I'll also link to this page for reference: https://docs.learnhouse.app/self-hosting/organization-hosting-modes

**Review by swve — COMMENTED** [review]
Hey @chrishollandaise, nice finds and LGTM 👍

You're right about the extra step—it doesn't add much value and can definitely be combined with the following one. I originally included it to demonstrate how org detection works based on the 'Hosting Modes', but we can merge the two steps and include a brief note explaining that the multi-org setup won't work locally.

I'll also link to this page for reference: https://docs.learnhouse.app/self-hosting/organization-hosting-modes

**Comment by chrishollandaise** [comment]
@swve Glad you agree. That totally makes sense, and I think that once multi-org setup works locally it'll be helpful to future contributors to have some more details.

File: pages/setup-dev-environment.mdx [hunk 1]
```diff
@@ -96,20 +104,27 @@ This will install all the dependencies needed for the frontend, and for this you
...
+Add an .env file in the frontend folder with the following content
+
+<Callout type="warning">
+  Setting MultiOrg to true won't work locally for now, please keep it false
+</Callout>
 
 ```env filename="apps/web/.env" {1-2} copy
 NEXT_PUBLIC_LEARNHOUSE_MULTI_ORG=false
-NEXT_PUBLIC_LEARNHOUSE_DEFAULT_ORG=test
+NEXT_PUBLIC_LEARNHOUSE_DEFAULT_ORG=defaultorg
```
Comment (swve) [inline pages/setup-dev-environment.mdx:115]:
```suggestion
NEXT_PUBLIC_LEARNHOUSE_DEFAULT_ORG=default
```
As that's the exact assigned slug when you first install the backend

**Review by swve — CHANGES_REQUESTED** [review]
One little change and i'll be good to merge

**Review by swve — APPROVED** [review]

**Comment by swve** [comment]
Thanks for your contribution @chrishollandaise 🙏

**Pull request merged**